sources/packages/web/src/assets/css/base.scss

```diff
@@ -378,3 +378,8 @@ b,
 strong {
   font-weight: $font-weight-bold;
 }
+
+// Added FontAwesome as fallback.
+.mdi:before {
+  font-family: "Material Design Icons", "FontAwesome", "BCSans", sans-serif !important;
```

Comment:
Can you please elaborate on this for my own knowledge only 😉

Reply:
ya... 
In our application, we have set icons default set as "mdi", 
![image](https://user-images.githubusercontent.com/77353155/221295165-b6e4d041-3edb-4123-b012-0e64d8a40f40.png)
so, vuetify component using the font awesome class is having class "mdi", which is the `Material Design` class and it's overriding the "FontAwesome" font family with "`Material Design` ". 
![image](https://user-images.githubusercontent.com/77353155/221291167-cfd25f90-94c4-4664-827c-49808d6b4363.png)

because of this, those icons are not visible,
eg, 
![image](https://user-images.githubusercontent.com/77353155/221291699-fd2bccab-94b8-4c31-a012-f2861d48a50c.png)
![image](https://user-images.githubusercontent.com/77353155/221291909-eaa677ad-b426-4165-a95c-4969b24fe53a.png)
![image](https://user-images.githubusercontent.com/77353155/221291967-6869a7e9-3787-47bf-af48-2fc1ca0b98db.png)

I tried to update the  `defaultSet`  with `fa`, 
![image](https://user-images.githubusercontent.com/77353155/221295388-336a1ce7-59f2-4792-ae3b-735e9556ff7d.png), doing so, `mdi` icons were having issues.
![image](https://user-images.githubusercontent.com/77353155/221295705-f34b73a1-39c9-4a7a-92c4-68345488a696.png)


I have updated the CSS, and I have reassigned the font awesome class with its family 
![image](https://user-images.githubusercontent.com/77353155/221295890-1864b9e5-d031-4abe-ada1-dc00bd3863d9.png)

Reply:
Ah, I see may be that is why the icons are not visible in v-select

![image](https://user-images.githubusercontent.com/54600590/221305106-5aa054b8-ef8d-42b6-9976-05e735ca80af.png)

Reply:
Thanks @ann-aot for providing the details.